index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                    <div class="row clearfix">
+                        <div class="col-md-2 column">
+                            <img class="img-thumbnail square-image" src="images/jensen_gao.png" alt="Jensen Gao"
+                                width="160" height="160">
+                            <h4>Jensen Gao</h4>
+                            <h5>Teaching Assistant</h5>
+                            <h5>OH: Fri 9 &ndash; 11am </h5>
+                            <h5>Location: Zoom (link on Canvas calendar)</h5>
+                        </div>
+                        <div class="col-md-2 column">
+                            <img class="img-thumbnail square-image" src="images/sri_jaladi.png" alt="Sri Jaladi"
+                                width="160" height="160">
+                            <h4>Sri Jaladi</h4>
+                            <h5>Teaching Assistant</h5>
+                            <h5>OH: Thurs 6 &ndash; 8pm </h5>
+                            <h5>Location: Zoom (link on Canvas calendar)</h5>
+                        </div>
+                        <div class="col-md-2 column">
+                            <img class="img-thumbnail square-image" src="images/daniel_shin.jpg" alt="Daniel Shin"
+                                width="160" height="160">
+                            <h4>Daniel Shin</h4>
+                            <h5>Teaching Assistant</h5>
+                            <h5>OH: Tues 5 &ndash; 7pm </h5>
+                            <h5>Location: Zoom (link on Canvas calendar) </h5>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="hrline">
+                <hr />
+            </div> <br>
+            <h3 id="topics">Timeline</h3>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th style="width: 12%">Date</th>
+                        <th style="width: 23%">Lecture</th>
+                        <th style="width: 15%">Deadlines</th>
+                        <th style="width: 50%">Notes</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr class="active">
+                        <td><b>Week 1</b><br />
+                            Wed, April 2
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Course intro + MDPs</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 1</b><br />
+                            Fri, April 4
+                        </td>
+                        <td><i><span class="label label-lecture text-base">TA Session</span></i>
+                            <a">Pytorch Tutorial</a>
```

Comment:
Lecture is earlier in the day than the tutorial, so this should come after lecture

Reply:
Same with the other tutorial